.github/workflows/tests.yml

```diff
@@ -107,6 +107,31 @@ jobs:
           name: bazel-testlogs
           path: bazel-testlogs
 
+  rules_apple_2_ios_integration_tests:
+    name: Build and Test ( Virtual Frameworks + rules_apple 2.x )
+    runs-on: macos-12
+    env:
+      USE_BAZEL_VERSION: 6.1.2
```

Comment:
Q: is this supposed to match `.bazelversion`?

Reply:
It is how `bazelisk` uses multiple versions different from the `.bazelversion`

Reply:
So no - `bazelisk` allows you to override the version which doesn't match the text file